src/main/java/jenkins/plugins/accurev/util/AccurevUtils.java

```diff
@@ -2,11 +2,11 @@
 
 import static org.apache.commons.lang.time.DateUtils.MILLIS_PER_SECOND;
 
-import edu.umd.cs.findbugs.annotations.CheckForNull;
 import hudson.FilePath;
 import hudson.model.Computer;
 import hudson.model.Node;
 import java.util.Date;
+import javax.annotation.CheckForNull;
```

Comment:
No no no... License issue and the whole JSR 305 is not in Java 8, I don't know the solution on then to keep using FindBugs annotation.

Reply:
Roger